user-docs/latest/content-manager/saving-and-publishing-content.md

```diff
@@ -0,0 +1,46 @@
+# Saving, publishing and deleting content
+
+Strapi allows you to manage your content throughout its whole lifecycle, whether your are working on its draft version, about to finish it and share it with the world, or wanting to delete it when it's obsolete.
+
+## Saving & publishing content
+
+::: warning IMPORTANT
+The possibility to manage drafts for contents comes from the Draft & Publish feature. This feature is activated by default, but it can be disactivated for any content type. If you disabled the Draft & Publish feature, saving your content means saving and publishing at the same time.
+:::
+
+Your contents can have 2 statuses: draft and published. You can see the current status indicated on the right of the interface, below the Information box.
```

Comment:
It's not really below the Information box, it's actually inside the Information box that you can have this information. Moreover, indeed you have an indication below that box, to let you know on which status you're making an edit on. 

I'm telling you this, as in the future with a versioning feature, we'll be able for instance, to edit a draft version while having a published content :)